### ISSUE TITLE
About '3x' in configs like 'atss_res101_coco_3x_800size.py'

### ISSUE THREAD

**Issue by Zhu-haow** [MegEngine/Models#117]
I wonder what the '3x' means in configs like 'atss_res101_coco_3x_800size.py' ?

Is 3x multi-scale training or multi-level feature ?

**Comment by wjfwzzc** [comment]
3x multi-scale training schedule, just like other methods.

**Comment by Zhu-haow** [comment]
> 3x multi-scale training schedule, just like other methods.

I cannot found definition code about 3x or 2x multi-scale training. Could you please help me point out the code? Or, what the difference about "atss_res101_coco_3x_800size.py" and "atss_resx101_coco_2x_800size.py" ?

**Comment by wjfwzzc** [comment]
Sorry for the above misleading description. More precisely, they're the learning rate schedule. `max_epoch` and `lr_decay_stages` are both different. The concept is proposed in [Rethinking ImageNet Pre-training
](https://arxiv.org/abs/1811.08883) and is widely used in many detection repos such as [Detectron2](https://github.com/facebookresearch/detectron2/blob/main/MODEL_ZOO.md).

**Comment by MegChai** [comment]
Maybe we should add more information in the docstring of some pre-trained models' definitions.

```
def atss_res101_coco_3x_800size(**kwargs):
    r"""
    ATSS trained from COCO dataset.
    `"ATSS" <https://arxiv.org/abs/1912.02424>`_
    `"FPN" <https://arxiv.org/abs/1612.03144>`_
    `"COCO" <https://arxiv.org/abs/1405.0312>`_
    """
```

Then users can get details by calling `megengine.hub.help("megengine/models", "atss_res101_coco_3x_800size")`

```shell
\n    ATSS trained from COCO dataset.\n    `"ATSS" <https://arxiv.org/abs/1912.02424>`_\n    `"FPN" <https://arxiv.org/abs/1612.03144>`_\n    `"COCO" <https://arxiv.org/abs/1405.0312>`_\n    '
```

The newline character does not seem to be escaped properly. 😓

**Comment by Zhu-haow** [comment]
Thanks for your reply.  I regarded 3x as multi image scales initially.
